README.md

```diff
@@ -228,7 +228,7 @@ Git section is consists with `git_branch` and `git_status` subsections. It is sh
 | `SPACESHIP_GIT_SHOW` | `true` | Show Git section |
 | `SPACESHIP_GIT_PREFIX` | `on ` | Prefix before Git section |
 | `SPACESHIP_GIT_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after Git section |
-| `SPACESHIP_GIT_SYMBOL` | ` ` | Character to be shown before Git section |
+| `SPACESHIP_GIT_SYMBOL` | ` ` | Character to be shown before Git section (Requires powerline patched font) |
```

Comment:
I like consistency, let's make _Requires_ lowercased, as it continues previous sentence.